I haven't had a chance to look yet, but there is an issue with multiplechoice
questions that prevents them from submitting correctly.

The reported error has this form:

Traceback (most recent call last):
  File "checker.py", line 91, in do_check
    **args)
  File "/catsoop/tutor.py", line 329, in _wrapped_func
    return orig(*args, **info)
  File "-data/_cached/__QTYPES__/multiplechoice/multiplechoice.py", line 54, in handle_submission
    sub = submissions[info['csq_name']]
KeyError: 'q000003'


The submitted value isn't actually being sent to the server, so this is a JS
problem.

